windows/src/desktop/help/common/os.md

```diff
@@ -15,6 +15,8 @@ Windows operating systems:
 
 -   Windows 10
 
+-   Windows 11
+
```

Comment:
Are we officially supporting Windows 11 in version 15?

Reply:
I thought we're officially supporting anything past Windows 7. You or @MakaraSok were able to run Keyman in a Windows 11 virtual box, right?

Reply:
Yes we do. I just thought early on @mcdurdin had said we would wait till version 16. Mind you that was when 15 was going to be released in March. We have had more time with some Beta testers on Win 11 now.
I just checked the latest acceptance test Makara was working on and he tested Win 11. So I am fine if @mcdurdin  is ok with that.

Reply:
From discussion in standup, we can have Keyman 15 mention support for Win 11.